front-end/src/app/tools/cash-on-hand-override/cash-on-hand-override.component.html

```diff
@@ -5,7 +5,7 @@ <h1>Line 6a override</h1>
     <div class="col-6 pl-0">
       <strong>WHAT IS A LINE 6A OVERRIDE?</strong>
       <p>
-        When there is a discrepancy between the reported (FECFile-calculated) cash on hand and the correct
+        When there is a discrepancy between the reported (FECfile+-calculated) cash on hand and the correct
```

Comment:
Probably and analyst question - I wonder if we want get rid of the hyphen to avoid the potentially confusing `FECfile+-calculated`

Reply:
Answered in comments from Jonella
https://fecgov.atlassian.net/browse/FECFILE-2214?focusedCommentId=35091
Language should be (calculated by FECfile+) and (calculated by the committee).